contents/blog/cant-make-friends-asynchronously.md

```diff
@@ -0,0 +1,46 @@
+---
+date: 2022-07-13
+title: "You can't make friends asynchronously"
```

Comment:
```suggestion
title: "How we learned remote companies can become "too asynchronous"
```

Reply:
Or juste "Remote companies can become too asynchronous"

Context: I think it's important we clarify we're talking about remote companies, as that's distinct from purely 'social' relationships between individuals.

Reply:
i was thinking about making it a bit more pithy ie:

"A cautionary tale of an all remote company being too asynchronous"

Reply:
I wouldn't overthink it, tbh.

It's a short, tight piece which makes a salient point about running a remote company. It's easy to overelaborate an opinion to the point it loses its impact. This piece works.